src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -160,20 +137,74 @@ private void OnGUI()
             // EditorGUILayout.EndToggleGroup();
         }
 
+        private void OnLostFocus()
+        {
+            Validate();
+
+            EditorUtility.SetDirty(Options);
+            AssetDatabase.SaveAssets();
```

Comment:
Should this be instead:

```suggestion
            if (Validate())
            {
                EditorUtility.SetDirty(Options);
                AssetDatabase.SaveAssets();
            }
```

Reply:
If the user closes the window we would throw away all their other changes just because the DSN is invalid.